src/lib/jetlore.js

```diff
@@ -0,0 +1,22 @@
+    return JL;
+}
+
+export default getJetlore();
```

Comment:
This is still causing a module-level side-effect. There should be no direct function calls at the module level if webpack is going to tree-shake correctly.